TestRandom.java

```diff
@@ -1,6 +1,23 @@
 // Tests the distribution generated by Java's Math.random() function.
 public  class  TestRandom {
 	public static void main(String[]  args) {
-	    // Replace this comment with your code
+	    int n=1;
```

Comment:
What about the user input that determines the number of times we call Math.random?